app/frontend/src/components/admin/AdminVersions.vue

```diff
@@ -79,6 +105,7 @@ export default {
         version: '',
         id: '',
       },
+      message: this.$t('trans.adminVersions.notificationMsg'),
```

Comment:
Is this variable really necessary? If we were to extend the functionality of this component, wouldn't this variable name have to be changed as well to be more descriptive? Why not just use the translation directly in the notification?